3rdparty/openvx/include/openvx_hal.hpp

```diff
@@ -2,6 +2,7 @@
 #define OPENCV_OPENVX_HAL_HPP_INCLUDED
 
 #include "opencv2/core/hal/interface.h"
+#include "opencv2/imgproc/hal/interface.h"
```

Comment:
Try to build OpenCV with disabled imgproc module:
```
cmake -DBUILD_opencv_imgproc=OFF .
```
=> include dirs from imgproc module will not be added
=> this include statement will fail